components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dto/OAuthConsumerAppDTO.java

```diff
@@ -300,5 +321,122 @@ public void setTokenBindingValidationEnabled(boolean tokenBindingValidationEnabl
 
         this.tokenBindingValidationEnabled = tokenBindingValidationEnabled;
     }
+
+    public String getTokenEndpointAuthMethod() {
+        return tokenEndpointAuthMethod;
+    }
+
+    public void setTokenEndpointAuthMethod(String tokenEndpointAuthMethod) {
+        this.tokenEndpointAuthMethod = tokenEndpointAuthMethod;
```

Comment:
Add a new line after the method definition. Check other places as well.

Reply:
fixed